src/ch/ch_process.c

```diff
@@ -608,20 +607,14 @@ chProcessNetworkPrepareDevices(virCHDriverPtr driver, virDomainObjPtr vm)
             // This info will be used while generating Network Json
             priv->tapfd = g_steal_pointer(&tapfd);
             priv->tapfdSize = tapfdSize;
-            priv->tapName = g_new(char, IFNAMSIZ);
-            if (virStrcpy(priv->tapName, net->ifname, IFNAMSIZ) < 0) {
-                virReportError(VIR_ERR_INTERNAL_ERROR,
-                       _("net->ifname %s too big for destination"), net->ifname);
-                goto cleanup;
-            }
          }
     }
 
     return 0;
 
     cleanup:
         g_free(tapfd);
-    return -1;
+    return 0;
```

Comment:
Can you change this to "return -1"? In case of a failure this should return -1 here.

Reply:
Done.